webapp_tracet/templates/trigger_app/proposal_form.html

```diff
@@ -223,8 +223,13 @@ <h2>Target Telescope Settings</h2>
                     newOption.appendChild(optionText);
                     // and option value
                     newOption.setAttribute('value',telescope);
+                    if (telescope === "{{form.event_telescope.value}}") {
```

Comment:
not an issue, just a question. What does a triple equals sign do in javascript?

Reply:
It's the same as == in other languages. Because javascript I guess